endpoints/schemas/InvoiceSchemas.yaml

```diff
@@ -0,0 +1,262 @@
+openapi: "3.0.0"
+info:
+  title: Invoice Schema
+  version: 4.0.0.0
+  description: Invoice Schema, Based on TIP 4.0.0 Working Draft
+  termsOfService: http://placeholderdomain.io/terms/,
+  contact: 
+    name: MIT
+    email: tip-mailing-list@tbd.com
+    url: http://placeholderdomain.io
+  license: 
+    name: MIT
+    url: https://opensource.org/licenses/MIT
+paths: {} 
+components: 
+  schemas: 
+    invoiceObject: 
+      required: 
+      - transactionHeader
+      - referenceId
+      - invoiceDate
+      - invoiceRevisionDate
+      - calendarType
+      - billingCycle
+      - startDate
+      - endDate
+      - buyerLocation
+      - advertiser
+      - buylineCashTrade
+      - grossAmount
+      - commision
+      - totalNetDollars
+      - unitType
+      - remittanceInfo
+      properties:
+        transactionHeader: 
+          $ref: 'commonSchemas.yaml#/components/schemas/transactionHeader'
+        referenceId:
+          description: Information grouping all reference ids and their source that are associated to the proposal
+          type: array # TODO any constraints?
```

Comment:
Array has no specified datatype for the items: https://swagger.io/docs/specification/data-models/data-types/#array